docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -250,18 +272,18 @@ jobs:
         type: string
         default: main
     environment:
-      SEMGREP_RULES: >- # more at semgrep.dev/explore
-        p/security-audit
-        p/secrets
+      # Scan with rules set in Semgrep App's rule board
+      # Get your token at semgrep.dev/orgs/-/settings/tokens
+      SEMGREP_APP_TOKEN: $SEMGREP_APP_TOKEN
 
       # Scan changed files in PRs, block on new issues only (existing issues ignored)
       SEMGREP_BASELINE_REF: << parameters.default_branch >>
 
     # == Optional settings in the `environment:` block
 
-    # Instead of `SEMGREP_RULES:`, use rules set in Semgrep App.
-    # Get your token from semgrep.dev/manage/settings.
-    #   SEMGREP_APP_TOKEN: $SEMGREP_APP_TOKEN
+    # Instead of `SEMGREP_APP_TOKEN:`, set hard-coded rulesets, 
```

Comment:
So this no longer uses rules set in Semgrep App? Maybe we should specify this information still somehow.

Reply:
yeah, see line 277. We're still using the Semgrep App, just by default now, rather than hard-coded by default.